### PR TITLE
chore: Update`possibleConfigFiles`

### PR DESCRIPTION
**Pull request by jacobdalamb**
## Changes

- What does this change?
- Be short and concise. Bullet points can help!
- Before/after screenshots can help as well.
- Don't forget a changeset! `pnpm exec changeset`

## Testing

n/a

## Docs

https://github.com/withastro/docs/pull/6093

### CODE REVIEW

**Review by lilnasy — APPROVED** [review]
Quick context for reviewers, `astro add` checks whether any one of `possibleConfigFiles` exists to determine whether to create a fresh one.

File: packages/astro/src/cli/add/index.ts [hunk 1]
```diff
@@ -160,6 +160,7 @@ export async function add(names: string[], { flags }: AddOptions) {
 					possibleConfigFiles: [
 						'./tailwind.config.cjs',
 						'./tailwind.config.mjs',
+						'./tailwind.config.ts',
```
Comment (bluwy) [inline packages/astro/src/cli/add/index.ts:163]: Should we also add .mts and .cts here?

Comment (florian-lefebvre) [reply packages/astro/src/cli/add/index.ts:163]: I think so since we already support cjs and mjs

**Review by ematipico — COMMENTED** [review]
A changeset is required here, would it be possible to add one?

File: packages/astro/src/cli/add/index.ts (hunk 1 above)
Comment (jacobdalamb) [reply packages/astro/src/cli/add/index.ts:163]: Added!

**Comment by jacobdalamb** [comment]
> A changeset is required here, would it be possible to add one?

Added!

**Review by bluwy — APPROVED** [review]

**Review by ematipico — APPROVED** [review]

**Pull request merged**